crowbar_framework/app/models/node_object.rb

```diff
@@ -74,7 +74,7 @@ def self.default_platform
   end
 
   def self.available_platforms
-    @@available_platforms ||= begin
+    @available_platforms ||= begin
```

Comment:
Are you sure that you have to remove the second ```@```? I think that is only the case for ```self << class```